docs/cumulusci_flow.rst

```diff
@@ -0,0 +1,273 @@
+This way, we can merge a fix and cut an emergency release at any time in the development process.
+
+To test that we can package main, we upload a beta release on every commit to main and then test that beta release in a variety of Salesforce org environments concurrently.
+This build ranges from 15 minutes to 2 hours depending on the project and a passing build is proof we can package main at any point in time.
+
+When the upload of the beta release is completed, the main branch is auto-merged_ into all open feature branches.
+New betas are published on Github as a Github Release along with automatically generated release notes created by parsing the body of all Pull Requests merged since the last production release
+
+
+
+CumulusCI and Main Builds
+^^^^^^^^^^^^^^^^^^^^^^^^^
+There are four main flows that facilitate main builds:
+
+* ``ci_master``: Deploys the main branch and all dependencies into the packaging org including incrementally deleting any metadata deleted in the commit.  The end result is a package that is ready to be uploaded from the packaging org.
+* ``release_beta``: Uploads a beta release of the code staged in the packaging org, creates a Github Tag and Release, generates release notes, adds them to the release, and merges main into all feature branches.
+* ``ci_beta``: Installs the beta and all dependencies into a fresh scratch org and runs the Apex tests.
+* ``ci_beta_install``: Installs the beta and all dependencies into a fresh scratch org. This is used to prepare environments for non-Apex testing such as automated browser tests.
+
+
+
+Tag Naming Convention
+^^^^^^^^^^^^^^^^^^^^^
+CumulusCI Flow uses two naming conventions for the tags generated by the process:
+
+* beta/1.2-Beta_3: Beta package releases
+* release/1.2:  Production package releases
+
+By differentiating beta and release tags, we allow tooling to query for the latest beta and the latest production release of each repository.
+
+
+
+CumulusCI and Tag Naming Convention
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+CumulusCI's default tag prefixes can be overridden if needed for particular projects by setting the values under project -> git:
+
+* **default_branch**: Override the default branch in the repository (default: ``main``, or the current branch during ``cci project init``)
+* **prefix_beta**: Override the prefix for beta tags (default: beta/)
+* **prefix_feature**: Override the prefix for feature branches (default: feature/)
+* **prefix_release**: Override the prefix for release tags (default: release/)
+
+
+
+Feature Branches
+----------------
+Like Github Flow, CumulusCI Flow uses a simple main/feature branch model.
+The main branch is the only permanent branch in the repository.
+All development work (features and bug fixes) is done in feature branches prefixed with ``feature/``.
+All commits on all feature branches are tested concurrently by our CI app, MetaCI, though any CI app could be used for the same purpose.
```

Comment:
I'm not sure we should be so concrete about MetaCI here. Maybe "via continuous integration, such as MetaCI or another solution"?